### PR TITLE
Add configuration value to redirect specific curated feeds to main feed

### PR DESCRIPTION
**Pull request by joelverhagen**
Progress on https://github.com/NuGet/NuGetGallery/issues/6472. Depends on https://github.com/NuGet/NuGetGallery/pull/6486.

If a curated feed name is in this new list, it will not even be checked for existence in the DB. It will just use all packages. I considered an HTTP redirect or calling the `ODataV2FeedController` but chose to take this approach since it has the best change of not breaking existing clients.

### CODE REVIEW

File: tests/NuGetGallery.Facts/Controllers/ODataFeedControllerFactsBase.cs [hunk 1]
```diff
@@ -217,7 +222,7 @@ private static ODataQueryContext CreateODataQueryContext<TFeedPackage>()
             return new ODataQueryContext(oDataModelBuilder.GetEdmModel(), typeof(TFeedPackage));
         }
 
-        private static async Task<dynamic> GetValueFromQueryResult<TEntity>(QueryResult<TEntity> queryResult)
+        protected static async Task<dynamic> GetValueFromQueryResult<TEntity>(QueryResult<TEntity> queryResult)
```
Comment (chenriksson) [inline tests/NuGetGallery.Facts/Controllers/ODataFeedControllerFactsBase.cs]: Is this change necessary?

Comment (joelverhagen) [reply tests/NuGetGallery.Facts/Controllers/ODataFeedControllerFactsBase.cs]: Nope, nice catch.
